frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -167,7 +167,10 @@ class Receive extends Component<Props, State> {
         const forceVerification = secureOutput.hasSecureOutput && !secureOutput.optional;
         const enableCopy = !forceVerification;
 
-        let address = receiveAddresses[activeIndex].address;
+        // TODO: unified-accounts, let user cycle through address types
+        const currentAddresses = receiveAddresses[0];
```

Comment:
I guess @thisconnect will take care of this.

Reply:
👍   yes, when it is merged into `staging-unified-accounts`